Vexiiriscv doesn't simulate

I just completely updated my installation and ran a basic command as a sanity check. Serv and vexriscv work but vexiiriscv doesn't:

```
$ litex_sim --cpu-type=vexiiriscv
```

And then I get a build error:

```
make: Entering directory '/home/pepijn/code/apicula/build/sim/gateware'
mkdir -p modules
make -C modules -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/Makefile
make[1]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules'
mkdir -p xgmii_ethernet
make MOD=xgmii_ethernet -C xgmii_ethernet -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/xgmii_ethernet/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/xgmii_ethernet'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/xgmii_ethernet/../.. -o xgmii_ethernet.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/xgmii_ethernet/xgmii_ethernet.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o tapcfg.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/tapcfg.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o taplog.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/taplog.c
cc -levent -shared -fPIC -lz -Wl,-soname,xgmii_ethernet.so -o xgmii_ethernet.so xgmii_ethernet.o tapcfg.o taplog.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/xgmii_ethernet'
cp xgmii_ethernet/xgmii_ethernet.so xgmii_ethernet.so
mkdir -p ethernet
make MOD=ethernet -C ethernet -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/ethernet/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/ethernet'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/ethernet/../.. -o ethernet.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/ethernet/ethernet.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o tapcfg.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/tapcfg.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o taplog.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/taplog.c
cc -levent -shared -fPIC -Wl,-soname,ethernet.so -o ethernet.so ethernet.o tapcfg.o taplog.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/ethernet'
cp ethernet/ethernet.so ethernet.so
mkdir -p serial2console
make MOD=serial2console -C serial2console -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2console/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/serial2console'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2console/../.. -o serial2console.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2console/serial2console.c
cc -levent -shared -fPIC -Wl,-soname,serial2console.so -o serial2console.so serial2console.o
rm serial2console.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/serial2console'
cp serial2console/serial2console.so serial2console.so
mkdir -p serial2tcp
make MOD=serial2tcp -C serial2tcp -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2tcp/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/serial2tcp'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2tcp/../.. -o serial2tcp.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/serial2tcp/serial2tcp.c
cc -levent -shared -fPIC -Wl,-soname,serial2tcp.so -o serial2tcp.so serial2tcp.o
rm serial2tcp.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/serial2tcp'
cp serial2tcp/serial2tcp.so serial2tcp.so
mkdir -p clocker
make MOD=clocker -C clocker -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/clocker/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/clocker'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/clocker/../.. -o clocker.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/clocker/clocker.c
cc -levent -shared -fPIC -Wl,-soname,clocker.so -o clocker.so clocker.o
rm clocker.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/clocker'
cp clocker/clocker.so clocker.so
mkdir -p spdeeprom
make MOD=spdeeprom -C spdeeprom -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/spdeeprom/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/spdeeprom'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/spdeeprom/../.. -o spdeeprom.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/spdeeprom/spdeeprom.c
cc -levent -shared -fPIC -Wl,-soname,spdeeprom.so -o spdeeprom.so spdeeprom.o
rm spdeeprom.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/spdeeprom'
cp spdeeprom/spdeeprom.so spdeeprom.so
mkdir -p gmii_ethernet
make MOD=gmii_ethernet -C gmii_ethernet -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/gmii_ethernet/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/gmii_ethernet'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/gmii_ethernet/../.. -o gmii_ethernet.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/gmii_ethernet/gmii_ethernet.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o tapcfg.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/tapcfg.c
cc -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/include -c -o taplog.o /home/pepijn/code/apicula/pythondata-misc-tapcfg/pythondata_misc_tapcfg/data/src/lib/taplog.c
cc -levent -shared -fPIC -lz -Wl,-soname,gmii_ethernet.so -o gmii_ethernet.so gmii_ethernet.o tapcfg.o taplog.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/gmii_ethernet'
cp gmii_ethernet/gmii_ethernet.so gmii_ethernet.so
mkdir -p jtagremote
make MOD=jtagremote -C jtagremote -f /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/jtagremote/Makefile
make[2]: Entering directory '/home/pepijn/code/apicula/build/sim/gateware/modules/jtagremote'
cc -c -Wall -O3 -ggdb -fPIC -Werror -I/home/pepijn/code/apicula/litex/litex/build/sim/core/modules/jtagremote/../.. -o jtagremote.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules/jtagremote/jtagremote.c
cc -levent -shared -fPIC -Wl,-soname,jtagremote.so -o jtagremote.so jtagremote.o
rm jtagremote.o
make[2]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules/jtagremote'
cp jtagremote/jtagremote.so jtagremote.so
make[1]: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware/modules'
mkdir -p /home/pepijn/code/apicula/build/sim/gateware/obj_dir
cc -c -ggdb -Wall -O3   -o /home/pepijn/code/apicula/build/sim/gateware/obj_dir/libdylib.o /home/pepijn/code/apicula/litex/litex/build/sim/core/libdylib.c
cc -c -ggdb -Wall -O3   -o /home/pepijn/code/apicula/build/sim/gateware/obj_dir/modules.o /home/pepijn/code/apicula/litex/litex/build/sim/core/modules.c
In file included from /home/pepijn/code/apicula/litex/litex/build/sim/core/modules.c:6:
In function ‘tinydir_readfile’,
    inlined from ‘litex_sim_load_ext_modules’ at /home/pepijn/code/apicula/litex/litex/build/sim/core/modules.c:68:14:
/home/pepijn/code/apicula/litex/litex/build/sim/core/tinydir.h:81:25: warning: ‘strcat’ accessing 4097 or more bytes at offsets 0 and 4096 may overlap 1 byte at offset 4096 [-Wrestrict]
   81 | #define _tinydir_strcat strcat
      |                         ^
/home/pepijn/code/apicula/litex/litex/build/sim/core/tinydir.h:532:9: note: in expansion of macro ‘_tinydir_strcat’
  532 |         _tinydir_strcat(file->path, file->name);
      |         ^~~~~~~~~~~~~~~
cc -c -ggdb -Wall -O3   -o /home/pepijn/code/apicula/build/sim/gateware/obj_dir/pads.o /home/pepijn/code/apicula/litex/litex/build/sim/core/pads.c
cc -c -ggdb -Wall -O3   -o /home/pepijn/code/apicula/build/sim/gateware/obj_dir/parse.o /home/pepijn/code/apicula/litex/litex/build/sim/core/parse.c
cc -c -ggdb -Wall -O3   -o /home/pepijn/code/apicula/build/sim/gateware/obj_dir/sim.o /home/pepijn/code/apicula/litex/litex/build/sim/core/sim.c
verilator -Wno-fatal -O3 --cc /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/Ram_1w_1rs_Generic.v --cc /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/Ram_1w_1ra_Generic.v --cc /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/VexiiRiscvLitex_e92c04213ef0ab400048eee18d35deed.v --cc /home/pepijn/code/apicula/build/sim/gateware/sim.v  --top-module sim --exe \
	-DPRINTF_COND=0 \
	sim_init.cpp /home/pepijn/code/apicula/litex/litex/build/sim/core/veril.cpp libdylib.o modules.o pads.o parse.o sim.o \
	--top-module sim \
	 \
	-CFLAGS "-ggdb -Wall -O3   -I/home/pepijn/code/apicula/litex/litex/build/sim/core" \
	-LDFLAGS "-lpthread -Wl,--no-as-needed -ljson-c -lz -lm -lstdc++ -Wl,--no-as-needed -ldl -levent " \
	--trace \
	 \
	 \
	--unroll-count 256 \
	--output-split 5000 \
	--output-split-cfuncs 500 \
	--output-split-ctrace 500 \
	 \
	-Wno-BLKANDNBLK \
	-Wno-WIDTH \
	-Wno-COMBDLY \
	-Wno-CASEINCOMPLETE \
	--relative-includes
%Error-PINNOTFOUND: /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/VexiiRiscvLitex_e92c04213ef0ab400048eee18d35deed.v:11554:6: Pin not found: 'rd_dataEn'
                                                                                                                                                                    : ... Suggested alternative: 'rd_data'
11554 |     .rd_dataEn (1'b1                                                ),  
      |      ^~~~~~~~~
                    ... For error description see https://verilator.org/warn/PINNOTFOUND?v=5.032
%Error-PINNOTFOUND: /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/VexiiRiscvLitex_e92c04213ef0ab400048eee18d35deed.v:11544:6: Parameter not found: 'rdLatency'
11544 |     .rdLatency      (1         )
      |      ^~~~~~~~~
%Error-PINNOTFOUND: /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/VexiiRiscvLitex_e92c04213ef0ab400048eee18d35deed.v:11579:6: Pin not found: 'rd_dataEn'
                                                                                                                                                                    : ... Suggested alternative: 'rd_data'
11579 |     .rd_dataEn (1'b1                                                ),  
      |      ^~~~~~~~~
%Error-PINNOTFOUND: /home/pepijn/code/apicula/pythondata-cpu-vexiiriscv/pythondata_cpu_vexiiriscv/verilog/VexiiRiscvLitex_e92c04213ef0ab400048eee18d35deed.v:11569:6: Parameter not found: 'rdLatency'
11569 |     .rdLatency      (1         )
      |      ^~~~~~~~~
%Error: Exiting due to 4 error(s)
        ... See the manual at https://verilator.org/verilator_doc.html for more assistance.
make: *** [/home/pepijn/code/apicula/litex/litex/build/sim/core/Makefile:44: sim] Error 1
make: Leaving directory '/home/pepijn/code/apicula/build/sim/gateware'
```

--- a/litex/soc/cores/cpu/vexiiriscv/core.py
+++ b/litex/soc/cores/cpu/vexiiriscv/core.py
@@ -156,7 +156,7 @@ class VexiiRiscv(CPU):
         vdir = get_data_mod("cpu", "vexiiriscv").data_location
         ndir = os.path.join(vdir, "ext", "VexiiRiscv")
 
-        NaxRiscv.git_setup("VexiiRiscv", ndir, "https://github.com/SpinalHDL/VexiiRiscv.git", "dev", "b4269ddc", args.update_repo)
+        NaxRiscv.git_setup("VexiiRiscv", ndir, "https://github.com/SpinalHDL/VexiiRiscv.git", "dev", "3282ca22", args.update_repo)
 
         if not args.cpu_variant:
             args.cpu_variant = "standard"

